### PR TITLE
Expand-the-Amazon-AWS-EC2-Integration

### PR DESCRIPTION
**Pull request by mihaitusa**
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173  Please review it when you can!

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package io.cloudslang.content.jclouds.entities;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by Mihai Tusa.
+ * 2/19/2016.
+ */
+public enum DiskTypes {
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java]: DiskType

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    DiskTypes(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public static String getDiskType(String input) {
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java]: are you using this?

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
+package io.cloudslang.content.jclouds.entities;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Created by Mihai Tusa.
+ * 2/22/2016.
+ */
+public enum InstanceTypes {
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java]: InstanceType

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+    // Storage optimized
+    HI1_4XLARGE("hi1.4xlarge"),
+    HS1_8XLARGE("hs1.8xlarge"),
+
+    // GPU instances
+    CG1_4XLARGE("cg1.4xlarge"),
+
+    // Micro instances
+    T1_MICRO("t1.micro");
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java]: wow

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java (hunk 2 above)
Comment (mihaitusa) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/DiskTypes.java]: @gituser173  It seems that I no longer need this... so I will remove it. Good catch!

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java (hunk 3 above)
Comment (mihaitusa) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/entities/InstanceTypes.java]: @gituser173  You'll right... I will rename it.

**Comment by mihaitusa** [comment]
We should close this PR because the gap will be filled migrating the existing OO Content op.

**Pull request closed without merging**